dask_mpi/cli.py

```diff
@@ -95,6 +97,14 @@ def main(
     name,
 ):
     comm = MPI.COMM_WORLD
+
+    world_size = comm.Get_size()
+    if scheduler and world_size < 2:
+        raise WorldTooSmallException(
+            f"Not enough MPI ranks to start cluster, found {world_size}, "
+            "needs at least 2 for the scheduler and at least 1 worker."
```

Comment:
The first time I read this my conclusion was "ah ok, so I need three" (2 + 1). But I think in reality we only need two right? If yes then I think "needs at least 2, one for the scheduler and a worker." would help with parsing.

Reply:
Yup good point. I've fixed this and updated the top comment.